storage/innobase/handler/ha_innodb.cc

```diff
@@ -18265,6 +18540,33 @@ static MYSQL_SYSVAR_BOOL(locking_fake_changes, srv_fake_changes_locks,
   "not take any locks at all.",
   NULL, NULL, TRUE);
 
+static MYSQL_SYSVAR_UINT(column_zlib_strategy, column_zip_zlib_strategy,
```

Comment:
Compressed column server variables completely reorganized. Only two are publicly accessible:
```c++
static MYSQL_SYSVAR_UINT(compressed_columns_zlib_level, srv_compressed_columns_zlib_level,
  PLUGIN_VAR_RQCMDARG,
  "Compression level used for compressed columns.  0 is no compression"
  ", 1 is fastest and 9 is best compression. Default is 6.",
  NULL, NULL, DEFAULT_COMPRESSION_LEVEL, 0, 9, 0);

static MYSQL_SYSVAR_ULONG(compressed_columns_threshold, srv_compressed_columns_threshold,
  PLUGIN_VAR_RQCMDARG,
  "Compress column data if its length exceeds this value. Default is 96",
  NULL, NULL, 96, 1, ~0UL, 0);
```
The rest are defined as private constants in `row0mysql.cc`:
```c++
const uint srv_compressed_columns_zlib_strategy = Z_DEFAULT_STRATEGY;
const bool srv_compressed_columns_zlib_wrap = true;
const bool srv_compressed_columns_zlib_use_heap = false;
```